### PR TITLE
test: Added test for file worflow in archive

### PR DESCRIPTION
**Pull request by Archana-PandeyM**
Test to verify that change in data of files in archive updates the host data in inventory when reuploaded. I have modified the test to run from integration test perspective instead of rewriting the existing test in iqe

### CODE REVIEW

File: integration-tests/test_file_workflow.py [hunk 1]
```diff
@@ -114,6 +118,34 @@ def test_file_workflow_with_an_archive_without_canonical_facts(
...
+    """
+    insights_client.register()
+    assert conftest.loop_until(lambda: insights_client.is_registered)
+    current_hostname = subprocess.check_output("hostname", shell=True).decode().strip()
+    try:
+        # set a new hostname
+        new_hostname = set_hostname()
+        # collect data and upload again
+        insights_client.run()
+        sleep(30)  # small wait for data to get reflected in inventory
```
Comment (m-horky) [inline integration-tests/test_file_workflow.py:141]: I'm gonna assume there's no better way to do this.

Comment (Archana-PandeyM) [reply integration-tests/test_file_workflow.py:141]: unfortunately yes there is no other way at this point then being hopeful(sleep).

**Review by m-horky — APPROVED** [review]
Internal CI failed on unrelated errors. LGTM.

**Pull request merged**